utils/maintenence.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# file: maintenence.sh
+# author: Matthew Gidden
```

Comment:
We don't typically put authorship in the files themselves as this should be covered by the authors file.